SnappyAPITests/CameraManagerTests.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  CameraManagerTests.swift
+//  Snappy
+//
+//  Created by Pawel Chmiel on 18.10.2016.
+//  Copyright © 2016 Droids On Roids. All rights reserved.
+//
+
+import XCTest
+import AVFoundation
+@testable import Snappy
+
+class CameraManagerTests: XCTestCase {
+    
+    var previewSize = CGSize(width: 200, height: 300)
+    override func setUp() {
```

Comment:
missing empty line between `previewSize` and `setUp` but I think the `setUp` & `tearDown` are not needed here as well so we can remove them both